app/lib/account/backend.dart

```diff
@@ -84,6 +84,28 @@ Future<User> requireAuthenticatedUser() async {
   return _authenticatedUser;
 }
 
+/// Authenticates with bearer [token] and populates [_authenticatedUser] with
+/// it, running in a new scope.
+///
+/// The [token] may be an oauth2 `access_token` or an openid-connect
+/// `id_token` (signed JWT).
+///
+/// When no associated User entry exists in Datastore, this method will create
+/// a new one. When the authenticated email of the user changes, the email
+/// field will be updated to the latest one.
+Future<R> withAuthorizationToken<R>(
```

Comment:
Should this be `@visibleForTestingOnly` ?

Reply:
It is also used from out handler chain.